libs/auth-api-lib/seeders/data/scope-activities-service-sessions.ts

```diff
@@ -0,0 +1,11 @@
+import { compose, createScope } from './helpers'
+
+export const up = compose(
+  createScope({
+    name: '@island.is/activities:read',
```

Comment:
We should maybe just rename to 
```suggestion
    name: '@island.is/activities',
```

Reply:
Renamed.